pitop/robotics/pan_tilt_controller.py

```diff
@@ -1,12 +1,143 @@
-from pitop import ServoMotor
-from pitop.system.port_manager import PortManager
+import configparser
+from os.path import (
+    exists,
+    isdir,
+    join,
+)
+from pathlib import Path
 
+from pitop.core.exceptions import UninitializedComponent
+from pitop.core.mixins import (
+    Stateful,
+    Recreatable,
+)
+from pitop.pma import ServoMotor
 
-class PanTiltController:
 
-    def __init__(self, servo_pan_port="S0", servo_tilt_port="S3"):
+class PanTiltController(Stateful, Recreatable):
+    CALIBRATION_FILE_DIR = ".config/pi-top/sdk"
+    CALIBRATION_FILE_NAME = "alex.conf"
```

Comment:
This is technically part of configuration. This should be reviewed alongside how we are intending to handle the JSON files, to ensure that we have a solid plan for file management moving forwards.